FRBDK/Glue/Glue/ViewModels/TabControlViewModel.cs

```diff
@@ -59,14 +63,14 @@ public void SetTabForCurrentType(PluginTabPage tab)
 
         public void ShowMostRecentTabFor(string typeName)
         {
-            if (Count < 2)
+            if (Count < 2 || typeName is null)
```

Comment:
I accidentally left this null check out on my last PR, which caused a crash when you click a funny spot on an item in the main tree view. I can make a separate PR for it if you'd prefer that.